yoti-stub/src/MockYotiSessionHandler.ts

```diff
@@ -105,7 +105,21 @@ class MockYotiSessionHandler implements LambdaInterface {
 			 case ResourcesEnum.INSTRUCTIONS:
 				 if (event.httpMethod === "PUT") {
 					 try {
-						 logger.info("Event received: PutInstructions", {event});
+					 	logger.info("Event received: PutInstructions", {event});
```

Comment:
could some screenshots be added please to the PR for both PUT and createSession

Reply:
Added :)